### PR TITLE
small non-views ruby i18n

### PR DESCRIPTION
**Pull request by yheuhtozr**
## What type of PR is this? (check all applicable)

- [X] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

I18n of relatively small independent ruby codes. Involving some deconstantization as needed.

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

- [ ] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [X] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: app/errors/authentication/errors.rb [hunk 1]
```diff
@@ -1,10 +1,6 @@
 module Authentication
   module Errors
-    PREVIOUSLY_SUSPENDED_MESSAGE = "It appears that your previous %<community_name>s " \
-                                   "account was suspended. As such, we've taken measures to prevent you from " \
-                                   "creating a new account with %<community_name>s and its community. If you " \
-                                   "think that there has been a mistake, please email us at %<community_email>s, " \
-                                   "and we will take another look.".freeze
+    PREVIOUSLY_SUSPENDED_MESSAGE = "".freeze
```
Comment (citizen428) [inline app/errors/authentication/errors.rb:3]: This surprises me a bit, why keep this constant around?

File: app/queries/articles/active_threads_query.rb [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
-      tags: ["discuss"],
-      time_ago: nil,
-      count: 10
-    }.with_indifferent_access.freeze
-
     MINIMUM_SCORE = -4
 
+    def self.default_options
+      {
+        tags: I18n.t("queries.articles.active_threads_query.default_tags").split(","),
```
Comment (citizen428) [inline app/queries/articles/active_threads_query.rb:7]: I'm not sure it makes sense to have `default_tags` translateable here. granted, hardcoding as we did before also seems less than ideal but I'd probably just leave a comment indicating that this needs to be looked at properly within the context of I18n.

File: app/services/users/create_mascot_account.rb [hunk 3]
```diff
@@ -1,29 +1,33 @@
...
-    MASCOT_PARAMS = {
-      email: "mascot@forem.com",
-      username: "mascot",
-      name: "Mascot",
-      profile_image: Settings::General.mascot_image_url,
-      confirmed_at: Time.current,
-      registered_at: Time.current,
-      password: SecureRandom.hex
-    }.freeze
+    MASCOT_PASSWORD = SecureRandom.hex.freeze
```
Comment (citizen428) [inline app/services/users/create_mascot_account.rb:3]: The only thing that changes is that `name` is now translatable. IMO you can keep the original constant around and change the newly added `mascot` method to this:

```ruby
def self.mascot
  @mascot ||= MASCOT_PARAMS.merge(I18n.t("create_mascot.name"))
end
```

Or if we want to keep the changes as minimal as possible, you reuse the existing `merge` call in `mascot_params`:

```ruby
MASCOT_PARAMS.merge(
  name: I18n.t("create_mascot.name")
  password_confirmation: MASCOT_PARAMS[:password])
)
```

That may actually be my preferred solution.

File: app/errors/authentication/errors.rb (hunk 1 above)
Comment (yheuhtozr) [reply app/errors/authentication/errors.rb:3]: No, good catch 😂

File: app/queries/articles/active_threads_query.rb (hunk 2 above)
Comment (yheuhtozr) [reply app/queries/articles/active_threads_query.rb:7]: Admittedly I'm not very sure about the design intention, but I thought that if the value is not hardcoded anywhere else, you may use it to let viewers from different locales focus on each localized tags with current available means only, if it makes sense.

Comment (citizen428) [reply app/queries/articles/active_threads_query.rb:7]: Gut feeling: it's hardcoded elsewhere. I'd probably leave it as a comment for now as mentioned above.

**Review by jeremyf — APPROVED** [review]

**Review by citizen428 — APPROVED** [review]

**Pull request merged**